src/engine/shared/config_variables.h

```diff
@@ -65,6 +65,7 @@ MACRO_CONFIG_INT(InfSoldierBombs, inf_soldier_bombs, 3, 1, 16, CFGFLAG_SERVER, "
 MACRO_CONFIG_INT(InfMercBombs, inf_merc_bombs, 15, 1, 100, CFGFLAG_SERVER, "Number of poison grenades for the mercenary")
 MACRO_CONFIG_INT(InfMercLove, inf_merc_love, 1, 0, 1, CFGFLAG_SERVER, "Enables love bombs for the mercenary (hammer)")
 MACRO_CONFIG_INT(InfBarrierLifeSpan, inf_barrier_lifespan, 30, 0, 100, CFGFLAG_SERVER, "Barrier lifespan")
+MACRO_CONFIG_INT(InfVoodooAliveTime, inf_voodoo_alive_time, 200, 0, 10000, CFGFLAG_SERVER, "How long a voodoo keeps staying alive after being killed (in ms)")
```

Comment:
what do you think about setting this to 800 by default?

Reply:
800 can reach pretty much any cover spot for engineers, i would release him weaker first. I'll settle for 550

![screenshot_2019-01-13_11-33-28](https://user-images.githubusercontent.com/6261556/51084346-25187000-1728-11e9-87f5-72f0adb1735f.png)